com.mandelbulber.Mandelbulber.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: com.mandelbulber.Mandelbulber
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: mandelbulber2
+rename-icon: mandelbulber
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/pfgconfig
+  - /share/doc
+  - /share/man
+  - "/bin/gsl*"
+  - "*.la"
+  - "*.a"
+
+modules:
+  - name: gsl
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.6.tar.gz
+        sha256: b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8
+
+  - name: opencl-headers
+    buildsystem: simple
+    build-commands:
+      - cp -av CL /app/include
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-Headers/archive/refs/tags/v2020.06.16.tar.gz
+        sha256: 2f5a60e5ac4b127650618c58a7e3b35a84dbf23c1a0ac72eb5e7baf221600e06
+  
+  - name: opencl-cpp-headers
+    buildsystem: cmake
+    config-opts:
+      - -DBUILD_DOCS:BOOL=OFF
+      - -DBUILD_TESTS:BOOL=OFF
+      - -DBUILD_TESTS:BOOL=OFF
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2.0.12.tar.gz
+        sha256: 20b28709ce74d3602f1a946d78a2024c1f6b0ef51358b9686612669897a58719
+
+  - name: lzo
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+  - name: mandelbulber
+    buildsystem: qmake
+    subdir: mandelbulber2/qmake
+    config-opts:
+      - mandelbulber-opencl.pro
+    post-install:
+      - install -Dm644 "${FLATPAK_BUILDER_BUILDDIR}/$FLATPAK_ID.metainfo.xml" "${FLATPAK_DEST}/share/metainfo/$FLATPAK_ID.metainfo.xml"
+      - install -Dm644 "${FLATPAK_BUILDER_BUILDDIR}/mandelbulber128.png" "${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/mandelbulber.png"
+      - install -Dm644 "${FLATPAK_BUILDER_BUILDDIR}/mandelbulber2/deploy/linux/mandelbulber2.desktop" "${FLATPAK_DEST}/share/applications/$FLATPAK_ID.desktop"
```

Comment:
This line shouldn't be needed.
Is `mandelbulber2` the name of the app or is it `mandelbulber`?
Because qmake install creates `$PREFIX/share/applications/mandelbulber2.desktop`, which would mean:
- appid is wrong and should be com.mandelbulber.Mandelbulber2
- appid is correct and desktop file should be renamed. This is done by using `rename-desktop-file: mandelbulber2.desktop`  (after _rename-icon_ is a good location).

Reply:
I think I might named it wrong, because I though that number 2 simply to indicate version, but I think in this case, it is part of the name.